spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-fabric8-client-reload/pom.xml

```diff
@@ -26,9 +26,12 @@
 			<artifactId>spring-boot-starter-actuator</artifactId>
 		</dependency>
 
+		<!-- not bootstrap starter, because we want to be able to disable bootstrap per-test -->
```

Comment:
Where are the tests that are not using bootstrap?

Reply:
Now I see things clearer.  For some reason before I was just looking at a single commit and not all the changes.  Sorry for the dumb question.